src/scripts/benefit-indicator.ts

```diff
@@ -227,11 +320,37 @@ async function getMetadata(api: D2Api): Promise<Metadata> {
                 },
                 filter: { id: { eq: DE_MINISTRY_VISIT_ID } },
             },
+            categoryCombos: {
+                fields: {
+                    id: true,
+                    name: true,
+                },
+                filter: {
+                    code: {
+                        eq: DEFAULT_CATEGORY_COMBO_NAME,
+                    },
+                },
+            },
+        })
+        .getData();
+
+    const d2DataSetResponse = await api
+        .request<{ dataSets: D2DataSet[] }>({
+            method: "get",
+            url: "/dataSets",
+            params: {
+                fields: "id,dataSetElements",
+                filter: `dataSetElements.dataElement.id:eq:${DE_MINISTRY_VISIT_ID}`,
+                paging: false,
+            },
```

Comment:
No need to change it, but what was the problem with api.models.dataSets or api.metadata.get({dataSets: ...}) ?

Reply:
I was trying to apply the filter using `metadata.get`, but typescript was complaining.

```
filter: {
  dataSetElements: {
    dataElement: {
      eq: DE_MINISTRY_VISIT_ID
    }
 }
}
```

Looks like we cannot add filter for nested fields?